eng/pipelines/aggregate-reports.yml

```diff
@@ -1,3 +1,12 @@
+resources:
+  pipelines:
+  - pipeline: net-core
+    project: internal
+    source: net - core
```

Comment:
Does this support the id number instead of name? If so that might be a little more stable if we rename this pipeline.

Reply:
I'll try that.

Reply:
using the pipeline Id does not work.